.travis.yml

```diff
@@ -3,4 +3,4 @@ python:
   - "3.7"
 install: pip install -r test_requirements.txt
 script:
-  ./bootstrap.py --bootstrap --all catdog && cd ../pulp_catdog && black --diff --check --exclude docs/conf.py .
+  ./plugin-template --bootstrap --all pulp_catdog catdog && cd ../pulp_catdog && black --diff --check --exclude docs/conf.py .
```

Comment:
Isn't `bootstrap` just one of the sections included in `--all`? It looks redundant(?)

Reply:
Good point. I'll update this.